packages/outbox-core/lib/outbox.ts

```diff
@@ -0,0 +1,170 @@
+import { AbstractPeriodicJob, type JobExecutionContext } from '@lokalise/background-jobs-common'
+import type { PeriodicJobDependencies } from '@lokalise/background-jobs-common/dist/periodic-jobs/periodicJobTypes'
```

Comment:
if we are missing some types there, it's preferable to adjust exports on the packages rather than use the dist import

Reply:
Sure, I didn't notice that. On it.